apps/rush-lib/src/cli/actions/UpgradeSelfAction.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as os from 'os';
+import colors from 'colors/safe';
```

Comment:
I've been trying to slowly move Rush to use the `@rushstack/node-core-library` `Terminal` API for logging, which has colors built in. Can you use that instead?